src/components/message-parts/link-attachments/LinkAttachment.js

```diff
@@ -267,46 +269,75 @@ export default class LinkAttachment extends PureComponent {
     )
   }
 
-  render() {
+  renderRemoveButton = () => {
+    const { isMenuOpened } = this.state
+
+    const isAllowedToRemove = this.props.onRemove && isMenuOpened
+
+    return (
+      isAllowedToRemove && (
+        <Menu {...this.menuProps} getContentNode={this.getContentNode} />
+      )
+    )
+  }
+
+  renderAttachments = () => {
     const {
       authorName,
       title,
       text,
       imageUrl,
-      thumbUrl,
       embedHtml,
       fields,
       footer,
-      className,
-      onRemove,
       classes,
     } = this.props
 
-    const { isMenuOpened } = this.state
-    const isAllowedToRemove = onRemove && isMenuOpened
+    return (
+      <div className={classes.main}>
+        {authorName && this.renderAuthor()}
+        {title && this.renderTitle()}
+        {text && this.renderText()}
+        {fields.length > 0 && this.renderFields()}
+        {footer && this.renderFooter()}
+        {imageUrl && !embedHtml && this.renderImage()}
+        {embedHtml && this.renderEmbed()}
+      </div>
+    )
+  }
+
+  render() {
+    const { imageUrl, embedHtml, thumbUrl } = this.props
+
+    const { className, messageText } = this.props
 
     return (
-      <div
-        onMouseEnter={this.onMouseEnter}
-        onMouseLeave={this.onMouseLeave}
-        ref={this.onRefContent}
-        className={className}
-      >
-        <Bubble hasArrow={false}>
-          {isAllowedToRemove && (
-            <Menu {...this.menuProps} getContentNode={this.getContentNode} />
-          )}
-          <div className={classes.main}>
-            {authorName && this.renderAuthor()}
-            {title && this.renderTitle()}
-            {text && this.renderText()}
-            {fields.length > 0 && this.renderFields()}
-            {footer && this.renderFooter()}
-            {imageUrl && !embedHtml && this.renderImage()}
-            {embedHtml && this.renderEmbed()}
+      <div ref={this.onRefContent} className={className}>
+        {messageText ? (
+          <Bubble
+            onMouseEnter={this.onMouseEnter}
+            onMouseLeave={this.onMouseLeave}
+            hasArrow={false}
+          >
+            {this.renderRemoveButton()}
+            {this.renderAttachments()}
+            {!imageUrl &&
+              !embedHtml &&
+              thumbUrl &&
+              this.renderImageLinkPreview()}
+          </Bubble>
+        ) : (
+          <div
+            onMouseEnter={this.onMouseEnter}
+            onMouseLeave={this.onMouseLeave}
+          >
+            {this.renderAttachments()}
+            {!imageUrl &&
+              !embedHtml &&
+              thumbUrl &&
+              this.renderImageLinkPreview()}
```

Comment:
I couldn't reproduce the case how to get to this part. How do I create a message to show `renderImageLinkPreview`?